versions.yaml

```diff
@@ -243,6 +243,12 @@ externals:
     url: "https://github.com/containerd/nydus-snapshotter"
     version: "v0.1.0"
 
+  ovmf:
```

Comment:
I'd like to ask you to add an subsection called sev under the newly introduced OVMF.  QEMU with TDX also uses OVMF and we can leverage this very same entry later on.

An example of how can be done is: https://github.com/kata-containers/kata-containers/blob/3f069c7acbfa3bd789255ca72f5b56414f4123e3/versions.yaml#L153-L160

Reply:
Sounds good, I have added a new SEV subsection. Glad to know this can be used for tdx in the future.